ShareFile/Plugin.ShareFile.Android/ShareFileImplementation.cs

```diff
@@ -98,15 +93,15 @@ private string WriteFile(string fileName, byte[] bytes)
 
             try
             {
-                var localFolder = Android.OS.Environment.GetExternalStoragePublicDirectory(Android.OS.Environment.DirectoryDownloads).AbsolutePath;
+                var localFolder = Application.Context.CacheDir.AbsolutePath;
```

Comment:
what is the reason for changing path?

Reply:
didn't find a way to configure ```fileprovider``` for this ```downloads``` folder path, it should be done pretty easily if it is critical using xml with paths

Reply:
How it worked before your changes?

Reply:
before changes, it worked directly with files, so it doesn't care about url, but using ```ContentProvider``` requires xml file with paths that should be used